README.md

```diff
@@ -0,0 +1,67 @@
+# DOGS BREEDS
+
+> Search for your favorite dog breed
+
+## Video 
+
+https://www.loom.com/share/cf74443662ab4455b24c608900185181
+
+## Built With
+
+- JavaScript
+- React
+- Redux
+- Git
+- Webpack
+- npm
+- CSS
+- Jest
+
+## Getting Started
+
+To get a local copy, follow these simple steps.
+
+### Prerequisites
+
+Web browser installed
+
+### Setup
+
+Cloning a repository
+
+- On GitHub.com, navigate to the main page of the repository;
+
+- Above the list of files, click "Code" button;
+
+- Copy the URL to clone the repository. 
+
+  - To clone the repository using HTTPS : `https://github.com/ElsonOtake/TVMaze.git`
+
+  - To clone the repository using an SSH key, including a certificate issued by your organization's SSH certificate authority : `git@github.com:ElsonOtake/TVMaze.git`
+
+  - To clone a repository using GitHub CLI : `gh repo clone ElsonOtake/TVMaze`
+
+- Open Terminal;
+
+- Change the current working directory to the location where you want the cloned directory;
+
+- Type `git clone`, and then paste the URL you copied earlier;
+
+- run `npm install` to install all dependencies;
+
+- run `npm run start` to start the application locally.
+
+
+## Authors
+
+👤 **Andrés Garzon**
+
+- GitHub: [@githubhandle](https://github.com/andgarzonmal)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/andres-garzon-maldonado-951a2a180/
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
```

Comment:
- [ ] Kindly add an "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

Reply:
- Well Resolved 👍🏽